CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ changed to just run docker commands in the Makefile. See `make docker-run` and
 - HAProxy plugin tag has changed from `host` to `server`
 - UDP output now supported
 - Telegraf will now compile on FreeBSD
+- Users can now specify outputs as lists, specifying multiple outputs of the
```

Comment:
You might like to add that this is a breaking change. Existing configs will cause Telegraf to exit, right?